cli/index.js

```diff
@@ -73,6 +419,10 @@ y.command({
         },
         password: {
             describe: 'Password of the new account'
+        },
+        Birthday:{

```

Comment:
Why is this one uppercase when everything else is lower case?

Reply:
done